app/assets/javascripts/admin.js

```diff
@@ -0,0 +1,2 @@
+//= require application
```

Comment:
do not require application js here, application and admin are two separate layouts and we should not mix them